deploy/clowdapp.yaml

```diff
@@ -35,10 +47,8 @@ objects:
               timeoutSeconds: 120
             readinessProbe:
               failureThreshold: 3
-              httpGet:
-                path: /
+              tcpSocket:
                 port: 8000
-                scheme: HTTP
```

Comment:
Is this change intentional? I would expect the readiness to be exactly like liveness.

Reply:
I've not been able to make the readiness probe working, but might have been another issue, let me try :)